src/users/cd-tickets.vue

```diff
@@ -39,10 +41,21 @@
         this.currentUser = res.body.user;
       },
       async loadEvents() {
-        this.usersDojos.forEach(async (dojo) => {
-          const res = await EventService.loadEvents(dojo.dojoId);
-          this.events = this.events.concat(res.body);
-        }, this);
+        const query = { status: 'published' };
+        query.afterDate = moment().unix();
+        query.utcOffset = moment().utcOffset();
+
+        return Promise.all(this.usersDojos.map(dojo =>
```

Comment:
I'd prefer
```javascript
return Promise.all(this.usersDojos.map(async (dojo) => {
  const events = await EventService.v3.get(dojo.dojoId, {
    params: {
      query,
      related: 'sessions.tickets',
    }
  });
  this.events = events
    .reduce((red, dojoEvents) => red.concat(dojoEvents.body.results), [])
    .sort(EventUtils.orderByStartTime);
}));
```

Reply:
Basically, I just want to avoid `.then` 😛 It just looks a bit cleaner to me.

Reply:
the then is after the promise.all. That would mean that we don't use a reducer but directly concat at each results. It could also work, i guess ?

Reply:
Ah, all the more reason to change it so. 😅